client/res/layout/settings.xml

```diff
@@ -55,9 +55,9 @@
 
         <EditTextPreference
                 android:defaultValue="192.168.0.100"
-                android:key="ip_address"
+                android:key="custom_endpoint"
```

Comment:
Don't change the preference key, you'll break any existing install. You can just change the label. Also, this isn't a URL, it is a domain. You may also want to update the summary.